### ISSUE TITLE
SetLogVerbosityLevel not working

### ISSUE THREAD

**Issue by dariooo512** [egramtel/tdsharp#31]
## Expected Behavior
Logs from the library are no longer getting displayed on the console.

## Actual Behavior
The API Crashes or displays an error in the console

## Steps to Reproduce the Problem
Case 1:
```
_client = new TdClient();
await _client.ExecuteAsync(new TdApi.SetLogVerbosityLevel { NewVerbosityLevel = 0 });
```
this yields:
![image](https://user-images.githubusercontent.com/8339146/71935787-02d4f900-31a8-11ea-9ce5-626561bb9f55.png)

Case 2:
Using syncronus method:
```
 _client = new TdClient();
 _client.Execute(new TdApi.SetLogVerbosityLevel { NewVerbosityLevel = 0 });
```
this yields an ArgumentNullException:
![image](https://user-images.githubusercontent.com/8339146/71935886-416ab380-31a8-11ea-94a3-1166cd305ac6.png)

## Specifications

  - Version: TdLib 1.4.0 (master version)
  - Platform: Win10
  - Runtime: Net Core 3.0

**Comment by x2bool** [comment]
Hi! I suggest upgrading to 1.5.x. Could you try 1.5 and the latest version from master?

**Comment by dariooo512** [comment]
I'm using the latest version updated on Jan 3 2020, which I'm reading from TDLib.csproj and TDLib.Api.csproj and is 1.4.0.

**Comment by x2bool** [comment]
Do you use TDLib.Native?

**Comment by dariooo512** [comment]
Yes, i'm preparing a sample to propose in a future PR on this repo. 

You can find the code [here](https://github.com/dariooo512/tdsharp/tree/master/Samples/TdLib.Samples.GetChats). 
To reproduce, add the keys in Program.cs and uncomment this line 
`//_client.Execute(new TdApi.SetLogVerbosityLevel { NewVerbosityLevel = 0 });`

**Comment by x2bool** [comment]
tdlib.native is currently on 1.3: https://github.com/ForNeVeR/tdlib.native/issues/23

This is likely to be the problem

**Comment by dariooo512** [comment]
1.5.0 is not currently available on nuget, i tried 1.4.0 preview 1 and the crash went away.

Some logs are still getting written before the SetLogVerbosityLevel is executed.
![image](https://user-images.githubusercontent.com/8339146/71995555-10878e80-323b-11ea-89a0-e0b081b0d1d6.png)

I guess this cannot be fixed due to the logs coming from the cpp code. I was wondering if there's something can be done.

Anyway that solved my problem.

**Comment by dariooo512** [comment]
Small update: i found out calling `TdLog.SetVerbosityLevel(0);` before instantiating the TdClient works even better, removing all logs without the need of SetLogVerbosityLevel.
